src/gui/KM_InterfaceGamePlay.pas

```diff
@@ -3810,6 +3777,14 @@ procedure TKMGamePlayInterface.UpdateStateIdle(aFrameTime: Cardinal);
 end;
 
 
+function TKMGamePlayInterface.IsDragScrollingAllowed: Boolean;
+begin
+  Result := not (gGame.IsPaused and (fUIMode in [umSP, umMP]))

```

Comment:
When overriding a methods, please always include "inherited". Even in this trivial case, when it is always True in parent class.